psi-ms.obo

```diff
@@ -23617,6 +23617,231 @@ relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_value_concept STATO:0000291 ! quantile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000163
+name: identified precursor intensity distribution mean
+def: "From the distribution of identified precursor intensities, the mean. The intensity distribution of the identified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000164
+name: unidentified precursor intensity distribution mean
+def: "From the distribution of unidentified precursor intensities, the mean. The intensity distribution of the unidentified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000165
+name: identified precursor intensity distribution sigma
+def: "From the distribution of identified precursor intensities, the sigma value" [PSI:MS]
+comment: The intensity distribution of the precursors informs about the dynamic range of the acquisition in relation to identifiability.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based
+relationship: has_value_concept STATO:0000237 ! standard deviation
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000166
+name: unidentified precursor intensity distribution sigma
+def: "From the distribution of unidentified precursor intensities, the sigma value" [PSI:MS]
+comment: The intensity distribution of the precursors informs about the dynamic range of the acquisition in relation to identifiability.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based
+relationship: has_value_concept STATO:0000237 ! standard deviation
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000167
+name: charged peptides ratio 1+ over 2+
+def: "Ratio of 1+ peptide count over 2+ peptide count in identified spectra" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000168
+name: charged spectra ratio 1+ over 2+
+def: "Ratio of 1+ spectra count over 2+ spectra count in all MS2" [PSI:MS]
```

Comment:
very close to `MS:4000063` (all you need to do is compute one division 'manually' from the table).
Should at least be mentioned as 'RELATED'.
Same for all the other "charged spectra ..." metrics